docs/src/emulation.md

```diff
@@ -3,35 +3,60 @@ id: emulation
 title: "Emulation"
 ---
 
```

Comment:
Sketch:
- intro - maybe expand on the feature list?
- Devices in the config + library tab with `newContext()`. Example should have "Desktop Chromium" + "iPhone 11".
- Viewport - included in the device, but you can override for some tests with `test.use()` + library tab with `page.setViewportSize()`.
- Locale and timezone - can set globally in the config, and override for some tests with `test.use()`. Snippet with test.use + library tab with `newContext()`.
- Permissions - config snippet + library tab.
- Geolocation is like locale.
- Color scheme and media is like locale.
- User agent is included in the device. If you really need to test a different user agent string, test.use() + library tab.
- javaScriptEnabled to test how your page behaves with no javascript. Example with test.use + library tab.